src/sentry/api/endpoints/sentry_app_details.py

```diff
@@ -41,6 +41,15 @@ def put(self, request, sentry_app):
             'overview': request.json_body.get('overview'),
         }
 
+        if data.get('events'):
+            if ('error' in data['events']) and not features.has('organizations:integrations-event-hooks',
+                                                                sentry_app.owner,
+                                                                actor=request.user):
```

Comment:
We should move this into a function called `has_event_hooks` or something to make it a little more concise in this function.